debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/CreateTableParserListener.java

```diff
@@ -31,7 +31,10 @@ public CreateTableParserListener(MySqlAntlrDdlParser parser, List<ParseTreeListe
     @Override
     public void enterColumnCreateTable(MySqlParser.ColumnCreateTableContext ctx) {
         TableId tableId = parser.parseQualifiedTableId(ctx.tableName().fullId());
-        if (parser.databaseTables().forTable(tableId) == null) {
+        Table existedTable = parser.databaseTables().forTable(tableId);
```

Comment:
@harveyyue I am really not sure if this is correct solution. How it is possible to have two {{CREATE TABLE}} for the same table? It sounds like a hack/workaround to a different problem.

Reply:
@jpechane Yes, there are more  {{CREATE TABLE}} statements for same table.
Our dba use precona tool to alter table schema, when alter table schema, will create the table `_some_table_new` first, in other word there are some `_some_table_new` created statements in history topic, and we add the regex expression to exclude these intermediate tables to avoid capturing duplicated data to intermediate topic, this filter will set null value to property `tableChanges` of HistoryRecord, so antlr has to parse same table for many times during the restarted connector phase.